### DISCUSSION TITLE
Is there gonna be something like postman.setNextRequest(null) but for flows?

### DISCUSSION THREAD

**Discussion by tomalazarov** [postmanlabs/postman-flows#211] [Q&A] [answered]
Hi guys :) As we all know postman.setNextRequest(null) works only in the collection runner. So what about flows? I'm curious how did you handle this. I have a solution but for sure it does not look so elegant, indeed the flow looks more and more complicated and like over rushed. This is what I do - if a request fails for some reason, let's say the services are down and we got anything else but not status code 200 > so in this case I don't want to continue with the execution of the requests afterwards > so after each Send Request block I'm adding a Condition block where I simply check if status code is 200 > if it is OK then the data is passed through and next request is fired, if there's an issue and the condition is not satisfied then it stops right there. 

What are your thoughts about that and are there any more clear solutions?
Postman team are you planning to implement something like postman.setNextRequest(null) but for flows?

**Comment by saswatds** [comment, marked as answer]
@tomalazarov As we are trying to establish flows as a visual language by itself, we are keeping ourselves clear of the collections way of doing things, i.e with javascript.  The usage of condition block is currently the correct way, but we could also add a simpler block which behave like a filter. 

I do agree that larger flows will start looking ugly, and this is where I think we will be putting our efforts next to reduce the UI clutter with grouping and collapsing. @samay-v This is a valid concern, so lets use this opportunity to understand root issue here and come up with a good scriptless solution.
